api.ts

```diff
@@ -364,7 +364,7 @@ export interface PackFormulaDef<ParamsT extends ParamDefs, ResultT extends PackF
 export interface StringFormulaDef<ParamsT extends ParamDefs> extends CommonPackFormulaDef<ParamsT> {
   execute(params: ParamValues<ParamsT>, context: ExecutionContext): Promise<string> | string;
   response?: {
-    schema: StringSchema;
+    schema: BaseStringSchema;
```

Comment:
How come this changed to BaseStringSchema, shouldn't it still be the StringSchema union type?